app/src/main/play/listings/ja-JP/full-description.txt

```diff
@@ -1,35 +1,41 @@
-このアプリは、URLリンクを開くための中間ステップとして機能します。
-リンクを開こうとすると、リンクをクリックし、このアプリを選んで開くと、そのリンクに関する情報をダイアログで表示され、確認・変更することができます。
-メールやソーシャルネットワークのアプリなど、外部リンクを開く必要がある場合に特に便利です。
+このアプリは、URLリンクを開くときのの中間ステップとして機能します。
+アプリでリンクを開くと、そのリンクに関する情報が表示され、確認・変更することができます。
+メールやSNSのアプリなどで、外部リンクを開く必要があるとき特に便利です。
 
 
-URLCheck は TrianguloY によって開発され、元々は個人的な使用のために開発されました。オープンソース(CC BY 4.0ライセンス)で、無料、広告・トラッカーなしです。
+URLCheck は TrianguloY によって開発され、元々は個人的な使用目的で開発されました。オープンソース(CC BY 4.0ライセンス)で、フリー、広告・トラッカーなしです。
 必要最小限の権限で動作します（インターネット権限は、モジュールチェックのために、ユーザが要求したときにのみ使用されます）。
 ソースコードは GitHub で公開されています。変更や修正を提案したり、新しい翻訳を提案したりしたい場合は、https://github.com/TrianguloY/UrlChecker をチェックしてください。
 
 
 このアプリはモジュール形式になっており、各モジュールの有効化・無効化・並べ替えが可能です。
 
-* テキストの表示: 現在の URL を表示します。自分で編集することもできます。無効にすることはできません。
+* URLの表示: 現在の URL を表示します。自分で編集することもできます。無効にすることはできません。
 
 * 履歴: 一般的な取り消し・やり直し機能です。モジュールを使った変更、ユーザーの編集などを表示し、元に戻すことができます。
 
-* ログ: チェックしたすべての URL のログを記録し、表示、編集、コピー、クリアなどができます。
+* ログ: チェックしたすべての URL のログを記録し、表示、編集、コピー、削除などができます。
 
-* ステータスコード: ボタンを押すと、その URL へのリクエストが行われ、ステータスコードが表示されます（OK、サーバーエラー、指定したページは存在しない、など）。さらに、それがリダイレクトリンクだった場合、ボタンを押すと、新しい URL を確認することもできます。（JavaScript を使ったリダイレクションは検出されません。URLは取得されますが、評価はされないためです）
+* ステータス コード: ボタンを押すと、その URL へのリクエストが行われ、ステータスコードが表示されます（OK、サーバーエラー、指定したページは存在しない、など）。さらに、それがリダイレクトリンクだった場合、ボタンを押すと、新しい URL を確認することもできます。（JavaScript を使ったリダイレクションは検出されません。URLは取得されますが、評価はされないためです）
 
 * URL スキャナー: VirusTotal を使用して URL をスキャンし、レポートを確認することができます。この機能を使用するには、無料の個人用 VirusTotal API キーが必要です。VirusTotal™ は、Google, Inc の商標です。
 
 * URL クリーナー: 「ClearURLs」のデータを使用して、URLからリファラルと不要なパラメータを削除できます。また、よく知られたURLのリダイレクトはオフラインで可能です。ClearURLsのデータは、https://docs.clearurls.xyz/latest/specs/rules/ で確認できます。
 
 * クエリの削除: デコードされた個々のURLのクエリを表示し、確認・削除することができます。
 
-* パターンモジュール。パターンに基づいてチェックを実行します。例えば: 
+* パターンモジュール。正規表現パターンで URL をチェックし、警告・提案・置換をします。パターンは変更したり、独自に作成したり、あるいは他のユーザーが作成したものを使用することもできます。
+内蔵のパターンは以下の通りです。
+    - ギリシャ文字のような非ASCII文字が含まれている場合に警告します。これはフィッシング詐欺に使われることがあるためです。例: Goõgle.com と Google.com など。
+    - http'を'https'に置き換えることを提案します。
+    - Youtube、Reddit、Twitter をプライバシーに配慮したフロントエンドに置き換えるよう提案します。 [デフォルトでは無効] 
+
+* パターンモジュール: パターンに基づいてチェックを実行します。例えば: 
     - HTTP のリンクを警告し、HTTPS に置き換えることができます。
     - ギリシャ文字のような非ASCII文字が使われていると、警告します。これはフィッシング詐欺に使われるためです。例: Goõgle.com と Google.com など。
```

Comment:
I think this is obsolete now and can be removed, the updated version is above, isn't it?